.changeset/many-snails-kneel.md

```diff
@@ -0,0 +1,6 @@
+---
+"@firebase/database": patch
```

Comment:
```suggestion

```

Reply:
Since there is no change in `@firebase/database`, we don't need to bump its version